config/features.yaml

```diff
@@ -61,3 +61,19 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+ip_address:
+        query_string: 'source_short:"LOG"'
+        attribute: 'ipv4'
+        store_as: 'ip_address'
+        re: '\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}'
+        aggregate: True
+        tags: ['IP address']
+
+username:
+        query_string: 'source_short:"LOG" AND body:"Invalid user"'
+        attribute: 'username'
+        store_as: 'username'
```

Comment:
Do we want to overwrite the username attribute here?

Or use the same as we have in other places of storing this as "extracted_username"